pytensor/scalar/basic.py

```diff
@@ -2696,7 +2698,7 @@ class Sign(UnaryScalarOp):
     nfunc_spec = ("sign", 1, 1)
 
     @staticmethod
-    def output_types_preference(x):
+    def _output_types_preference(x):
```

Comment:
```suggestion
    def _get_output_types_preference(x):
```

Since it's a method in this case, not a member variable

Reply:
This is what init of the base class will look for if I don't pass a function

Reply:
I don't get it

Reply:
Check for `ScalaeLoop.__init__`, this plays a role there